eventstore/lib.go

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
This file (the d-bus client API) should be in it's own package in this repo (`eventclient` or something) so that we don't have clients perhaps accessing the DB manipulation code.